src/components/widgets/JiraUser.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import { computed, toValue } from 'vue';
+
+const props = defineProps<{
+  displayName: string;
+  emailAddress?: string;
+  name?: string;
+  query?: string;
+}>();
+
+const formatedUser = computed(() => {
+  let output = props.displayName;
+  if (props.emailAddress) {
+    output += ` - ${props.emailAddress}`;
+  }
+  if (props.name) {
+    output += ` (${props.name})`;
+  }
+  if (props.query) {
+    output = output.replace(new RegExp(toValue(props.query), 'gi'), '<b>$&</b>');
```

Comment:
Is the use of a RegExp here in place of a string to filter out multiple occurrences?

Reply:
Yes, and also to keep the original case of the text.
For example, given this output: `Alvaro Tinoco - atinoco@email.com (atinoco)`, the query `tinoco` will transform to `Alvaro <b>Tinoco</b> - a<b>tinoco</b>@email.com (a<b>tinoco</b>)` (note the **uppercase T** on my last name), using a string the casing will be lost

Reply:
Nice, thank you for the context